### ISSUE TITLE
Improve design and sizing of search results slideshow view

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/dlme#1370]
The search results slideshow view is currently a bit dated in terms of styling and font/icon sizes. It also seems like it could take up more of the user's viewport and provide a potentially larger image to view (which is important here because that's the main point of the slideshow view: to see the image larger than with list or gallery view):

<img width="1496" alt="Screen Shot 2021-12-13 at 5 40 06 PM" src="https://user-images.githubusercontent.com/101482/145911761-3dae5f05-b3d0-47ec-84f2-9937a6d37a72.png">

I can work on a more complete ticket with design suggestions if we decide this should be part of the next workcycle, but the things I'm suggesting could be improved include:

- Increase font sizes of the item title and item counter
- Increase size of icons
- Use more of the available viewport so the image has the potential to be larger

We did a roughly similar update to Spotlight's uploaded item modal window earlier this year so there might be some patterns to follow there.

Note that this would be work on the `blacklight-gallery` gem, so would benefit any project that uses [that gem](https://github.com/projectblacklight/blacklight-gallery).

**Comment by mwerla** [comment]
Regarding the priority of improvements for the slideshow feature, I checked the usage based on data from June till November (December data looked a bit suspicious - too much test traffic I guess). 

Slideshow view makes:
- 3% of the search results browsing (URL contains /library/catalog?...)  (avg time on page 01:48)
- 7% of the predefined categories browsing (URL contains /library/browse/...  (avg time on page 01:45)

The other non-default option, gallery view makes:
- 22% of the search results browsing (URL contains /library/catalog?...) (avg time on page 01:33)
- 8% of the predefined categories browsing (URL contains /library/browse/...  (avg time on page 02:17)

So, taking into account that it's one of three options possible, for sure it's the least used one. But the usage for browsing predefined categories is not that small. Moreover, I also checked that for both types of pages usage of slideshow view is somehow associated with a longer avg time on page for a few tens of seconds.

So, to sum up I think it makes sense to work on that feature improvements. And I agree for sure with the three improvement areas suggested by @ggeisler.

**Comment by asmaaalkuwari** [comment]
I agree with both @mwerla and @ggeisler for the suggestion of the updated design for the slideshow view of the research results that can enhance the experience of the user. One suggestion as well that can be added to the fix is to update the icon design of the slideshow option. The immediate response when seeing the icon is (video) which might confuse the user (unless they hover over the icon and find the word slideshow pop up). to keep it aligned with the other icons as well and consistent, here are some suggested icons that can be added
![slideshow icon 2](https://user-images.githubusercontent.com/62650194/150092156-766a421f-3eae-4c37-b271-6ca39e90c0ad.png)
![slideshow icon](https://user-images.githubusercontent.com/62650194/150092163-0e0dd52d-9d5d-40ee-ba8a-ebd04a3d6036.png)
:

**Comment by ggeisler** [comment]
Thanks for the icon update suggestion @asmaaalkuwari. I think this is a valid point (the existing icon was selected quite a while back when there were fewer icon options available). For various reasons I'm not sure either of the two options you suggested are ideal (e.g., possibly the second but the first one has an outline, which none of the other result view icons next to it do, so I think will look incongruous). But I've created a [separate issue](https://github.com/sul-dlss/dlme/issues/1394) and added the design-needed label to it so I can look into possibilities when I have time.

**Comment by ggeisler** [comment]
Below is a mockup and more complete description of suggested visual design improvements to the slideshow view.

Note that in my original ticket comment I suggested we should be using a larger modal to display the image at a larger size; looking closer I now realize we're doing that part correctly. It's just that DLME has many small/thumbnail sized images so the modal is often small, but it's showing the image at full-size, so that part is good.

Things I do think we want to update:

<img width="938" alt="Screen Shot 2022-01-24 at 4 27 18 PM" src="https://user-images.githubusercontent.com/101482/150883107-07c7c42a-e282-489b-89f0-3b2729c30e60.png">

- **A:** Increase the size of the modal close button to `2.5rem` (currently `1.875rem`). I think we might also want to reduce the padding-top (or padding top and bottom) to `0.5rem` from `1rem` after increasing its size (to minimize empty space above the image).

- **B:** Increase font size of `.caption` to `1rem`. The current size is `14px`, which was Bootstrap's default in earlier versions, but is too small for readability. (We'll still need to override this in DLME, because our DLME font `Markazi` is a very small font and we redefine our base font size to be `1.25rem` to get a body font of `20px`, which is what we probably want here.)

- **C:** Similar to above, increase font size of `.counter` to `1rem`. 

- **D:** Increase size of pause and play SVG icons to 32x32. These are currently 24x24, which is a small display size and also results in a button that is doesn't meet minimal target size ([44x44](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html)).

The end result of those changes should look roughly similar to this (differences are small but will be more apparent when we also update DLME to increase the font size of caption and counter, as noted below):

<img width="938" alt="Screen Shot 2022-01-24 at 4 31 37 PM" src="https://user-images.githubusercontent.com/101482/150883260-61885609-3682-4a71-9883-35e3ded5086b.png">

After the changes above are done in Blacklight-gallery, we should -- in DLME -- override the `.caption` and `.counter` font-size to be `$font-size-base`, which will improve the readability of those elements when using the Markazi font in DLME.

**Comment by mwerla** [comment]
I think that the updates will make the view more readable for sure. I guess it's more useful for visual content than for small thumbnails of book covers, but that's just a matter of the content that we have in DLME :-) I have no further comments.

**Comment by thatbudakguy** [comment]
changes are now live in production

**Comment by thatbudakguy** [comment]
@ggeisler [via slack](https://stanfordlib.slack.com/archives/C5V53K69G/p1644435297546609?thread_ts=1644434678.522569&cid=C5V53K69G):
> Yeah, I looked at all the slideshow updates when making my “looks good” comment earlier this morning, but I guess that was on the separate ticket for the click target. Anyway, all looks good to me!

**Comment by mwerla** [comment]
@jacobthill No comments on the implemented design, but there is still no way to open the record page from the slideshow view, which makes it quite useless. But maybe that's a separate task?
